test/e2e/scheduling/predicates.go

```diff
@@ -836,11 +836,13 @@ func initPausePod(f *framework.Framework, conf pausePodConfig) *v1.Pod {
 				},
 			},
 			Tolerations:                   conf.Tolerations,
-			NodeName:                      conf.NodeName,
 			PriorityClassName:             conf.PriorityClassName,
 			TerminationGracePeriodSeconds: &gracePeriod,
 		},
 	}
+	if len(conf.NodeName) != 0 {
```

Comment:
why do we want this to go through the scheduler?

Reply:
To not block the main release, this acts more as a workaround. So I'm extracting this section into a separate commit and leave a TODO so as to buy us some time to find out the real root cause in the future.